### ISSUE TITLE
Not working

### ISSUE THREAD

**Issue by budliter** [plasticrake/homebridge-tplink-smarthome#164]
I have been using this for months and suddenly it has stopped working completely.  Here is the troubleshooting that I have done:
1.  The TPlink / KASA app works in controlling the lights - no issues with the lights or kasa control itself
2.  Homekit seems to be working - it knows which lights should be there and they show up on the app...I also reset the appletv that is running homekit to check
3.  The TPlink switch also show up in the homebridge gui as assesscories...but I can't make the lights turn on and off in this app

Any thoughts?

**Comment by hadiwina** [comment]
I suddenly started having exactly this issue as well. It seems that the Kasa app switches/plugs are working fine outside of Homebridge/Homekit. Tried controlling using the Home app doesn't actually affect the switch/plugs.

I tried removing and then reinstalling the plugin. But now, after the plugin reinstall, none of the accessories can be discovered (not showing up in the list of accessories in Homebridge > Accessories).

The only thing that I remember is that I did a firmware update on the switches and plugs earlier today. I wonder if that what the issue is.
